docs/developers/content-creation/creating-a-weapon.md

```diff
@@ -1,3 +1,120 @@
 # Creating a Weapon
 
-Some Text
+When starting content creation for gmod, creating a basic weapon using the SWEP structure is one of the easiest things you can do.
+For this guide I'll be using the Visual Studio Code with the glualint extension as referenced in the "Creating an addon" section of these documents.
+
+## File Format
+
+Firstly you need to make sure that you're using the correct folder structure.
+Your addon folder must be in this format:
+
+* `weapon_ttt_<weapon name>\gamemodes\terrortown\entities\weapons\weapon_ttt_\<weapon name>.lua`
+
+Your .vmt and .vtf icon files (what gets shown in the buy menu) will need to be stored in this format:
+
+* `\weapon_ttt_<weapon name>\materials\vgui\ttt\<weapon name>_icon.vtf`
+* `\weapon_ttt_<weapon name>\materials\vgui\ttt\<weapon name>_icon.vmt`
+
+See the [Creating An Addon](https://docs.ttt2.neoxult.de/developers/basics/creating-an-addon/) page for more info on folder structure
```

Comment:
i would suggest using a relative path instead of an absolute path. Also you can directly link to the exact section you are refering to :) 
```suggestion
See the [Creating An Addon](../../basics/creating-an-addon/#project-structure) page for more info on folder structure.
```